bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -48,6 +48,7 @@ The Account bridge has the following configuration elements:
 | clientId          | String | Yes      | Client ID provided for the application you created on http://dev.netatmo.com/createapp                                 |
 | clientSecret      | String | Yes      | Client Secret provided for the application you created                                                                 |
 | webHookUrl        | String | No       | Protocol, public IP and port to access openHAB server from Internet                                                    |
+| webHookPostfix    | String | No       | String appended to the generated webhook address                                                                       |
```

Comment:
Maybe you could mention in the description (here and in the i18n property) that the piostfix must start with a /

Reply:
Seemed obvious, but you're correct, it's better explained.

Reply:
Not so obvious as I will have used naturally a / while you say now it must be a \

Reply:
Are you sure of \ ? That looks strange for  HTTP URL.

Reply:
Getting too old I guess. Good luck you're there as the door keeper. Happy new year @lolodomo

Reply:
Happy new year too.